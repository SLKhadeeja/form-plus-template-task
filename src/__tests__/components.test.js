import React from "react";
import renderer from "react-test-renderer";
import Loader from "../components/loader";
import TemplateCard from "../components/templateCard";

it("renders correctly", () => {
  const templateCard = renderer.create(<TemplateCard name="A special template" describe="fake template description for test" />).toJSON();
  const loader = renderer.create(<Loader />).toJSON();
  expect(templateCard).toMatchSnapshot();
  expect(loader).toMatchSnapshot();
});