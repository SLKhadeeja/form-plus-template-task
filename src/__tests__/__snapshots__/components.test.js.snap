// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="card-container"
>
  <div
    className="card-info"
  >
    <h3>
      A special template
    </h3>
    <p />
  </div>
  <div
    className="use-template-container"
  >
    <p>
      Use Template
    </p>
  </div>
</div>
`;

exports[`renders correctly 2`] = `
<div
  id="outer"
>
  <div
    id="middle"
  >
    <div
      id="inner"
    />
  </div>
</div>
`;
